### ISSUE TITLE
How to build?

### ISSUE THREAD

**Issue by exikyut** [udem-dlteam/mimosa#91]
Found this repo by way of `ribbit`, which floated to the top of HN recently.

The screenshot made me curious :) so I decided to take a single-pass crack at running the quickstart.

This unfortunately failed at

```
Missing the Gambit archive... Downloading a default working version.
--2021-12-14 02:37:38--  http://www-ens.iro.umontreal.ca/~yvonsamu/gsc.exe
Resolving www-ens.iro.umontreal.ca (www-ens.iro.umontreal.ca)... 132.204.26.158
Connecting to www-ens.iro.umontreal.ca (www-ens.iro.umontreal.ca)|132.204.26.158|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-12-14 02:37:38 ERROR 404: Not Found.
```

And the Web Archive doesn't seem to have a copy of that URL. The README mentions a customized Gambit version, so I'd expect a stock version is unlikely to work. The ubuntu-6 VM you mention also seems to have disappeared, so I can't rebuild (?) either. (Which sounds reasonably involved in any case.)

TL;DR: I'm distantly curious about this, much like I've been about Scheme for a while now - to be entirely honest my fascination was piqued by the tiny fonts :D. I'm mostly writing this issue for other similarly-curious folk who may potentially come across the repo and want to give it a go.

Cheers!

**Comment by SamuelYvon** [comment]
Hey! Thanks for looking at this project!

Admittedly, I haven't built it in a bit, but I can take a look. I had restriction on my account on ` http://www-ens.iro.umontreal.ca/~yvonsamu/` so I could not keep the archives hosted there forever. 

I'll setup things and update the instructions and notify through this thread. If I seem to have forgotten, don't be afraid to `@`me :)

**Comment by florianfmmartin** [comment]
I think you forgot @SamuelYvon ;)

**Comment by mateodif** [comment]
@SamuelYvon I tried building it today and I'm having the same issue :) Just letting you know since you asked for it!

**Comment by florianfmmartin** [comment]
hi @mateodif i used @SamuelYvon 's fork of gambit scheme here: https://github.com/SamuelYvon/gambit, i directly moved the binary to ./archived-items/gambit/bin

**Comment by SamuelYvon** [comment]
Yes, you have to use my fork of gambit; otherwise it won't build. I don't remember the reasons exactly, but since it was highly coupled with the bindings with the OS (obviously) it might be some tweaks to some syscalls. Now that I think about it more, I thinks some syscalls were abstracted by a smaller subset.

@florianfmmartin If you can confirm everything worked, I can clarify the instructions in the README.

**Comment by florianfmmartin** [comment]
@SamuelYvon yes it did build with your fork
